app/Http/Middleware/RestrictAdministrationAccess.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class RestrictAdministrationAccess
```

Comment:
There is a better "Laravel-way" to do this. See https://laravel.com/docs/9.x/authorization#gates